packages/docs/content/blog/release-9.0.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Release 9.0
+date: 2024-01-25
+intro: Differentiating between labels, hints, and errors, standardizing dialog close buttons, improving drawers and dialogs, and refining typography design tokens.
+---
+
+import dialogXClose from '../../src/images/dialog-x-close.png';
+
+Along with a host of bugfixes and minor improvements, this major release focused on differentiating between labels, hints, and errors, standardizing dialog close buttons, improving drawers and dialogs, and refining typography design tokens. This release is scheduled for late January, 2024 and is [currently available in beta form](https://github.com/CMSgov/design-system/releases). Read the sections below for more details about this release.
+
+## Whatʼs new in 9.0
+
+### Differentiating between labels, hints, and inline errors
+
+Weʼve restructured form-field labels to improve accessibility, customizability, and clarity of guidance. [Weʼve explained this change in more detail in this migration guide](/migration-guides/v9-label-updates), but the gist is that labels used to contain hints and inline-error messages and now theyʼre separate components. The React components for the various design system form fields handle this change automatically, as they all have labels, hints, and error text built in. Making them separate elements makes them easier to style predictably. Moving the hint and error text out of the HTML `<label>` elements makes them an appropriate place to put interactive content, because assistive technology expects labels to be free of interactive elements. Not having a place to put interactive help resources for a field was a problem, and application teams had no path clear forward. We hope these changes can be leveraged to create even more helpful and accessible forms for delivering important government services.
+
+### Standardizing dialog close buttons
+
+Dialog close buttons have gotten a visual refresh. All themes have standardized on an "X" for close, which is a familiar part of the digital interface vernacular that is succinct and compact.
+
+<div className="ds-u-measure--wide ds-u-text-align--center ds-u-margin-top--3">
+  <img src={dialogXClose} width={600} className="ds-u-display--inline-block" alt="Screenshots showing the X close button in the dialog in unfocused and focused modes" />
+</div>
+
+This refreshed "X" button has a large hit area to make sure it is still easy to tap and click, and it still provides the same amount of context for screen-readers as before.
+
+### Improving drawers and dialogs
+
+We fixed [a focus-management bug in drawers and dialogs](https://github.com/CMSgov/design-system/pull/2890), but it required a small API change to the React components. Now instead of conditionally rendering them to show or hide, you pass an `isOpen` prop to control their visibility. The old API will be supported until the next major version, but a drawer or dialog will still have the bug until it is updated.
```

Comment:
Should we bold "The old API will be supported until the next major version" to make sure it stands out to folks skimming the post?

Reply:
Good idea